src/chainparams.cpp

```diff
@@ -214,10 +214,10 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_V20].nFalloffCoeff = 5;            // this corresponds to 10 periods
 
         consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].bit = 10;
-        consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].nStartTime = 19999999999;   // TODO: To be determined later
-        consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT;
+        consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].nStartTime = 1704088800;   // January 1, 2024
+        consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].nTimeout = 1767247200; // January 1, 2026
```

Comment:
why 2 years? `Wed Jan 01 2025 00:00:00 GMT+0000` -> `1735689600`

Reply:
not clear how soon platform will be deployed. 20.1 release some day since now (3-12 months later) -> activation will happen only after.

maybe 18 month makes more sense, but 2 year are legit for me